### PR TITLE
#164069175 Setup react router version 4

### PR DESCRIPTION
**Pull request by Peace-Apple**
## What does this PR do?

This PR installs and configures react router version 4 (react-router-dom)

## Description of the task to be completed

- Install react-router-dom
- Add dummy route for login and / to test the configuration
- Create a components and views file structure

## How should this be manually tested?

- Clone this repository
- cd into the directory Ah-aquaman-frontend
- Check out the branch ch-setup-react-router-164069175
- Run npm install to install all dependencies
- Run npm start to get the server running
- You will view the home page at `localhost/`
- Then you can view the login page at `localhost/login`

## What are the relevant pivotal tracker stories?

[#164069175](https://www.pivotaltracker.com/story/show/164069175)

## Screenshots

You can view the homepage at `localhost/`

image

Then the login page at `localhost/login`

image

### CODE REVIEW

**Review by wasibani-roy — APPROVED** [review]
Good work @Peace-Apple all looks good i can route to another page

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import React, { Component } from "react";
+import React from "react";
 import "./assets/css/App.scss";
+import Routes from "./routes";
 
 const App = () => (
```
Comment (CryceTruly) [inline src/App.js:5]: Thanks for the great work @Peace-Apple , However, the indenting here is causing issues with eslint,
You could write this block as 
```
const App = () => (
  <div>
    <Routes />
  </div>
);
```

**Review by CryceTruly — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (Peace-Apple) [reply src/App.js:5]: Okay @CryceTruly let me look into that

Comment (Peace-Apple) [reply src/App.js:5]: @CryceTruly  I have fixed it now

**Review by CryceTruly — APPROVED** [review]

**Review by AmosWels — CHANGES_REQUESTED** [review]
GoodWork @Peace-Apple, please create one folder called `auth` and place both the `home` and `login` component in that folder. 

Remember to use camel casing when naming folders. instead of `Home`, can name it `home`, instead of `HomeView`, use `homeView`

**Comment by Peace-Apple** [comment]
> GoodWork @Peace-Apple, please create one folder called `auth` and place both the `home` and `login` component in that folder.
> 
> Remember to use camel casing when naming folders. instead of `Home`, can name it `home`, instead of `HomeView`, use `homeView`

Okay thanks @AmosWels , let me work on that

**Comment by Peace-Apple** [comment]
> GoodWork @Peace-Apple, please create one folder called `auth` and place both the `home` and `login` component in that folder.
> 
> Remember to use camel casing when naming folders. instead of `Home`, can name it `home`, instead of `HomeView`, use `homeView`

I have worked on that @AmosWels

**Comment by CryceTruly** [comment]
Thanks, @Peace-Apple, The deployment works fine now.

**Review by AmosWels — APPROVED** [review]

**Pull request merged**